src/app/genes/gene-similar/gene-similar.component.ts

```diff
@@ -371,8 +376,16 @@ export class GeneSimilarComponent implements OnInit {
     customSort(event: SortEvent) {
         event.data.sort((data1, data2) => {
             let result = null;
-            const value1 = data1[event.field];
-            const value2 = data2[event.field];
+            let value1 = null;
+            let value2 = null;
+
+            if ('hgnc_symbol' === event.field) {
+                value1 = data1.hgnc_symbol || data1.ensembl_gene_id;
```

Comment:
If hgnc_symbol really is null or undefined when not set, I recommend using the nullish coalescing operator (??) instead of OR (||) when implementing a default value.  The empty string (""), 0, and false are also [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)), and in some contexts these are valid values.

Reply:
Good point!